deliver/lib/deliver/upload_metadata.rb

```diff
@@ -183,6 +195,11 @@ def verify_available_languages!(options)
         end
       end
 
+      # Reject "default" language from getting enabled
```

Comment:
@ohayon There is no "default" language on ITC, is there?? ITC kept erroring out saying "default is not a valid language" so I assumed the "default" was a `fastlane` only thing